.config.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
   // the defaults for these need to be pointing to prod
   API_URL: process.env.API_URL || 'https://api.tidepool.org',
   UPLOAD_URL: process.env.UPLOAD_URL || 'https://uploads.tidepool.org',
+  DATASERVICES_URL: process.env.DATASERVICES_URL || 'https://dataservices.tidepool.org',
```

Comment:
I believe this needs to be updated, according to @darinkrauss's response in Slack, no?